recipes/dcase2021_task4_separation_SED/local/utils.py

```diff
@@ -0,0 +1,57 @@
+import scipy
+import pandas as pd
+from pathlib import Path
+import os
+from desed.utils.evaluation_measures import compute_sed_eval_metrics
+
+
+def batched_decode_preds(
+    strong_preds, filenames, encoder, threshold=0.5, median_filter=7
```

Comment:
median_filter is not set in the functions, so we always 7 (it matches the 0.45 s as long as the features are computed like this, so ok, but maybe we have to be careful for later expes)

Reply:
I ll add this in the config yaml